packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -105,32 +106,34 @@ export default function MemberInvitationModal(props: MemberInvitationModalProps)
             <Controller
               name="role"
               control={newMemberFormMethods.control}
-              defaultValue={options[0]}
-              render={({ field: { onChange } }) => (
+              defaultValue={options[1]}
+              render={({ field: { option, onChange } }) => (
```

Comment:
```suggestion
              render={({ field: { value, onChange } }) => (
```
Not 100% sure if this is the solution to the type error but Im not sure option exists here

Reply:
fixing it